content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -0,0 +1,148 @@
+
+For example, you might create an extension that uses the `model` and `client` component types.
+
+</TopBlock>
+
+## About Prisma Client extensions
+
+When you use a Prisma Client extension, you create an _extended client_. An extended client is a lightweight variant of the standard Prisma Client that is wrapped by one or more extensions. The standard client does not change. You can add as many extended clients as you want to your project. [Learn more about extended clients](#extended-clients).
+
+You can associate a single extension, or multiple extensions, with an extended client. [Learn more about multiple extensions](#multiple-extensions).
+
+You can [share your Prisma Client extensions](/concepts/components/prisma-client/client-extensions/shared-extensions) with other Prisma users, and [import Prisma Client extensions developed by other users](#install-a-packaged-extension) into your Prisma project.
+
+### Extended clients
+
+Extended clients interact with each other, and with the standard client, as follows:
+
+- Each extended client operates independently in an isolated instance.
+- Extended clients cannot conflict with each other, or with the standard client.
+- All extended clients and the standard client communicate with the same [Prisma query engine](/concepts/components/prisma-engines/query-engine).
+- All extended clients and the standard client share the same connection pool.
+
+### Example use cases for extended clients
+
+Because extended clients operate in isolated instances, they are a good way to do the following:
+
+- Implement row-level security (RLS), where each HTTP request has its own client with its own RLS extension, customized with session data. This can keep each user entirely separate, each in a separate client.
+- Add a `user.current()` method for the `User` model to get the currently logged-in user.
+- Enable more verbose logging for requests if a debug cookie is set.
+- Attach a unique request id to all logs so that you can correlate them later, for example to help you analyze the operations that Prisma Client carries out.
+- Remove a `delete` method from models unless the application calls the admin endpoint and the user has the necessary privileges.
+
+## Extensions prerequisites
+
+Prisma Client extensions are available in Preview from version [4.7.0](https://github.com/prisma/prisma/releases/tag/4.7.0).
+
+To use Prisma Client extensions, you must enable the `clientExtensions` preview feature flag in the `generator` block of your `schema.prisma` file, as follows:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["clientExtensions"]
+}
+```
+
+## Create an extension
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an extended client. In your `$extends` structure, use the appropriate extension component or components (`model`, `client`, `result` or `query`).
```

Comment:
```suggestion
Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an extended client. In your `$extends` structure, use the appropriate extension component or components ([`model`](/concepts/components/prisma-client/client-extensions/model), [`client`](/concepts/components/prisma-client/client-extensions/client), [`result`](/concepts/components/prisma-client/client-extensions/result) or [`query`](/concepts/components/prisma-client/client-extensions/query)).
```

Reply:
I wouldn't normally repeat links in a logical section of the docs, but I think you're right here: users might skip down to this section, so it would be helpful to have these links here.